onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -1608,9 +1609,23 @@ bool GemmOpBuilder::IsQuantizedOp(const NodeUnit& node_unit) const {
 void GemmOpBuilder::AddInitializersToSkip(ModelBuilder& model_builder, const NodeUnit& node_unit) const {
   const auto& inputs = node_unit.Inputs();
   if (IsQuantizedOp(node_unit)) {
-    AddQuantizationScaleAndZeroPointToSkip(model_builder, *inputs[0].quant_param);               // b_scale, b_zp
-    AddInputToSkip(model_builder, inputs[1]);                                                    // b, b_scale, b_zp
-    AddQuantizationScaleAndZeroPointToSkip(model_builder, *node_unit.Outputs()[0].quant_param);  // y_scale, y_zp
+    if (node_unit.OpType() == "QlinearMatMul") {
+      AddQuantizationScaleAndZeroPointToSkip(model_builder, *inputs[0].quant_param);               // a_scale, a_zp
+      AddInputToSkip(model_builder, inputs[1]);                                                    // b, b_scale, b_zp
+      AddQuantizationScaleAndZeroPointToSkip(model_builder, *node_unit.Outputs()[0].quant_param);  // y_scale, y_zp
+    }
+    if (node_unit.OpType() == "Gemm") {                                               // QDQGemm
+      AddQuantizationScaleAndZeroPointToSkip(model_builder, *inputs[0].quant_param);  // x_scale, x_zp
```

Comment:
nit: readability is better if there's a blank line between unconnected statements. that way you can quickly skim between them without having to think about whether this is directly related to the `if (node_unit.OpType() == "QlinearMatMul") {` or not.

Reply:
Of course this could also be an `else if` to potentially save a string comparison, as OpType can't match both QLinearMatMul and Gemm.